apps/builder/app/builder/features/workspace/canvas-tools/outline/label.tsx

```diff
@@ -80,7 +81,8 @@ type LabelProps = {
 
 export const Label = ({ instance, instanceRect }: LabelProps) => {
   const [labelRef, position] = useLabelPosition(instanceRect);
-  const meta = getComponentMeta(instance?.component);
+  const metas = useStore(registeredComponentMetasStore);
+  const meta = metas.get(instance.component);
   if (meta === undefined) {
     return <></>;
```

Comment:
just curious why we use empty fragments instead of null

Reply:
Because jxx element or node do not accept them